script/check-caption-repo

```diff
@@ -1,29 +1,21 @@
 #!/usr/bin/env bash
+#
+# Check Caption Repo
 
-################################################################################
-############################ Check Caption Repo ################################
-################################################################################
-
-###########################################################
-# NOTE:                                                   #
-# You must have a githubteacher token                     #
-# saved to your ENV variables in order to use this script #
-###########################################################
+#################################################################
+# NOTE: You must have a personal access token (PAT)             #
+# saved to your environment variables to use this script.       #
+# We recommend a dedicated service account (e.g. githubteacher) #
+#################################################################
 
 COLLAB_REPO=$1
 
-################################################################################
```

Comment:
Thank you for cleaning these up @parkerbxyz.  This is the first thing I noticed when pulling down the code.  Now, we're back to code that's more easily readable because you can see more of it on a screen at the same time.